src/data-structures/linked-list/LinkedList.js

```diff
@@ -190,10 +190,17 @@ export default class LinkedList {
     return nodes;
   }
 
+  /**
+   * @param {Array} Array of nodes
```

Comment:
Please provide correct type doc definition. It should look something like: `@param {LinkedListNode[]} array - Array of nodes`.